src/common/htmlFileReader.ts

```diff
@@ -38,6 +39,26 @@ export default class HtmlFileReader {
         }
     }
 
+    /**
+     * Downloads the binary blob from the blob path
+     * @param asset The asset to download
+     */
+    public static async getAssetBlob(asset: IAsset): Promise<Blob> {
```

Comment:
Need tests.

Reply:
@wbreza ^ are tests coming in this PR or coming PR?

Reply:
this one - just notes to myself :)